hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/resources/hdfs-rbf-default.xml

```diff
@@ -49,13 +49,39 @@
   </property>
 
   <property>
-    <name>dfs.federation.router.rpc.async.enable</name>
+    <name>dfs.federation.router.async.rpc.enable</name>
     <value>false</value>
     <description>
       If true, router will process the RPC request asynchronously.
     </description>
   </property>
 
+  <property>
+    <name>dfs.federation.router.async.rpc.ns.handler.count</name>
+    <value>nsPlaceholder1:0,nsPlaceholder2:0</value>
```

Comment:
suggest to leave empty here because it is not valid value. Thanks.

Reply:
fixed. Thanks.